plugins/ccsecret.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ccsecret
+spec:
+  platforms:
+  - uri: https://github.com/rewanth1997/kubectl-ccsecret/releases/download/v1.0.0/kubectl-ccsecret-Darwin-x86_64.tar.gz
+    sha256: e22ab5c443dd1ae00fdc8e1a68f142edbde642bbc1c84862569a9c510b81b373
+    bin: kubectl-ccsecret
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-ccsecret/releases/download/v1.0.0/kubectl-ccsecret-Linux-x86_64.tar.gz
+    sha256: d215f0304a2bbc4dba1b1ec51d1718f3c9f708ea984f34bee7838df240e755f6
+    bin: kubectl-ccsecret
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-ccsecret/releases/download/v1.0.0/kubectl-ccsecret-Windows-x86_64.zip
+    sha256: 5d5469b792354010b4c9260d181a3ef8446b744f16481bcf447b4c81f7929865
+    bin: kubectl-ccsecret.exe
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: v1.0.0
+  homepage: https://github.com/rewanth1997/kubectl-ccsecret
+  shortDescription: Create kubectl secrets by taking sensitive input from console
+  description: |
+    kubectl-ccsecret allows users to create kubectl secrets by taking sensitive
+    input from the console to prevent information leakages via terminal history,
+    shoulder surfing, etc
+
+    Examples:
+
+    Create generic secret with --from-literal:
+    $ kubectl ccsecret generic my-secret --from-literal key1 --from-literal key2
+    Enter value for key1: ******
+    Enter value for key2: ******
```

```suggestion
    $ kubectl ccsecret generic my-secret --from-literal key
    Enter value for key: ******
```
I think the second item does not add much. But you can also leave it there.

---

True. Done.